# Single cell papers with code

Single cell papers with code can not only facilitate the reproducibility of biomedical researches, but also promote our skills of analyzing single cell data. 


'Papers with code' here means that authors provide necessary codes to reproduce figures or results in their papers.




### Visualization

- [[code]](https://github.com/faryabib/NatMethods_TooManyCells_analysis) Schwartz GW, Zhou Y, Petrovic J, Fasolino M, Xu L, Shaffer SM, et al. **TooManyCells identifies and visualizes relationships of single-cell clades.** Nat Methods 2020. https://doi.org/10.1038/s41592-020-0748-5.

### Tutorial

- [[code | Python]](https://github.com/theislab/single-cell-tutorial) Luecken, M.D., and Theis, F.J. (2019). **Current best practices in single‐cell RNA‐seq analysis: a tutorial.** Mol. Syst. Biol. 15, e8746.

### Trajectory analysis

- [[code | Python]](https://github.com/rajewsky-lab/planarian_lineages) Plass, M., Solana, J., Alexander Wolf, F., Ayoub, S., Misios, A., Glažar, P., Obermayer, B., Theis, F.J., Kocks, C., and Rajewsky, N. (2018). **Cell type atlas and lineage tree of a whole complex animal by single-cell transcriptomics.** Science (80-. ). 360.

### Cell Atlas

- [[code | R]](https://github.com/Single-Cell-Genomics-Group-CNAG-CRG/Tumor-Immune-Cell-Atlas) Nieto, P., Marc Elosua-Bayes, M., Trincado, J.L., Marchese, D., Massoni-Badosa, R., Salvany, M., Henriques, A., Mereu, E., Moutinho, C., Ruiz, S., et al. (2020). **A Single-Cell Tumor Immune Atlas for Precision Oncology.** BioRxiv 2020.10.26.354829.

- [[code | Python]](https://github.com/czbiohub/tabula-muris-senis) Tabula Muris Consortium. **A single-cell transcriptomic atlas characterizes ageing tissues in the mouse.** Nature 2020;583. https://doi.org/10.1038/s41586-020-2496-1.


### Developmental biology

- [[code | R]](https://github.com/ZornLab/Single-cell-transcriptomics-reveals-a-signaling-roadmap-coordinating-endoderm-and-mesoderm-lineage) Han, L., Chaturvedi, P., Kishimoto, K., Koike, H., Nasr, T., Iwasawa, K., Giesbrecht, K., Witcher, P.C., Eicher, A., Haines, L., et al. (2020). **Single cell transcriptomics identifies a signaling network coordinating endoderm and mesoderm diversification during foregut organogenesis.** Nat. Commun. 11.


### Experimental research

- [[code | R]](https://github.com/tgen/banovichlab) Habermann A, Gutierrez A, Bui L, Yahn S, Winters N, Calvi C, et al. **Single-cell RNA sequencing reveals profibrotic roles of distinct epithelial and mesenchymal lineages in pulmonary fibrosis.** Sci Adv 2020;6:eaba1972. https://doi.org/10.1101/753806.


- [[code | R]](https://github.com/marzamKI/neurogenic_astros) Zamboni, M., Llorens-Bobadilla, E., Magnusson, J.P., and Frisén, J. (2020). **A Widespread Neurogenic Potential of Neocortical Astrocytes Is Induced by Injury.** Cell Stem Cell 1–13.


- [[code | Python]](https://github.com/linnarsson-lab/ipynb-lamanno2016) La Manno, G., Gyllborg, D., Codeluppi, S., Nishimura, K., Salto, C., Zeisel, A., Borm, L.E., Stott, S.R.W., Toledo, E.M., Villaescusa, J.C., et al. (2016). **Molecular Diversity of Midbrain Development in Mouse, Human, and Stem Cells.** Cell 167, 566-580.e19.


